### DISCUSSION TITLE
Contribute by adding [New Task] to the Courses

### DISCUSSION THREAD

**Discussion by axyut** [b0sc/college#30] [General]
Since we do not have and are not entitled all the ideas in the world, we'll be accepting task addition for the courses based on your ideas, `what do you think students should practice to gain the practical understanding of theoretical concepts?`
- Your task idea must be relevant to the course.
- Your task must have a proper documentation on how it should be accomplished under the `[course_name]/README.md` file's Task Section.

### Start Discussion by providing following Information to Us.
- Context
  - For Course : Example: Dsa
  - Idea/Problem: Example: Solve Tower of Hanoi using recursion
- How will it help students grasp the Practical Understanding of CS concepts?

> This discussion exists because we do not want to waste the time you've put in opening the PR when we do not like the idea of adding that task in the repo. So discuss here before working on the idea/task so that maintainers agree.

**Comment by Upendra48** [comment]
I want to add a new task which is a **Problem Statement**

# Context

- **For Course**: Data Structures and Algorithms (DSA)
- **Idea/Problem**: Time Tracker

## How It Will Help Students Grasp the Practical Understanding of CS Concepts

This problem gives students a hands-on opportunity to apply data structures like **arrays**, **hash maps** (or dictionaries), and basic algorithms to solve a real-world scenario. By working through this example, students will gain a practical understanding of:


### Frequency Counting and Summation
   - This exercise requires calculating total usage times across multiple instances, reinforcing counting and summation operations using efficient data structures, a fundamental concept for analytics.

**Reply by axyut** [reply]
sure we can implement this.
